Feliz.Recharts/Tooltip.fs

```diff
@@ -12,6 +12,27 @@ type tooltip =
     static member inline offset (value: int) = Interop.mkTooltipAttr "offset" value
     static member inline filterNull (value: bool) = Interop.mkTooltipAttr "filterNull" value
     static member inline cursor (value: bool) = Interop.mkTooltipAttr "cursor" value
+    /// Specifies how the text in the tooltip should be formatted 
+    /// 
+    /// ie: tooltip.formatter (fun value _ _ -> $"{value}%%")
+    /// ie: tooltip.formatter (fun _ _ _ -> data), as long as "data" is a string
+    static member inline formatter(f: 'a->string->'b->'c) = Interop.mkTooltipAttr "formatter" (System.Func<_,_,_,_> f)
+    static member inline wrapperStyle (properties: #IStyleAttribute list) = Interop.mkTooltipAttr "wrapperStyle" (createObj !!properties)
+    static member inline itemStyle (properties: #IStyleAttribute list) = Interop.mkTooltipAttr "itemStyle" (createObj !!properties)
+    static member inline labelStyle (properties: #IStyleAttribute list) = Interop.mkTooltipAttr "labelStyle" (createObj !!properties)
+    /// The position of the tooltip in relation to the chart.
+    ///
+    /// Takes a tuple of (x : int, y : int)
+    /// Default value depends on the position of the mouse.
+    static member inline position (?x: int, ?y: int) = 
```

Comment:
what do you think about this instead:
```fs
static member inline position (?x: int, ?y: int) = 
  let position = createObj [
    "x" ==> defaultArg x 0
    "y" ==> defaultArg y 0
  ]

  Interop.mkTooltipAttr "position" position
```

Reply:
Looks good to me!

Reply:
If you want to avoid Fable magic you can even use anonymous record.